frontend/public/components/_quota.scss

```diff
@@ -0,0 +1,7 @@
+.co-m-resource-quota-icon {
+  padding-left: 34px;
+}
+
+.co-m-resource-quota-empty {
+  font-size: 15px;
```

Comment:
Curious why we need a special rule here.

Reply:
Because we are mixing `fa` and `pf` icons, this is needed so that `fa` icons are center aligned with the `pf` icons

Reply:
It seems like there should be a `pficon-resources-empty` so we don't have to mix in the font awesome icon for this one special case.

FYI @serenamarie125 there are

* `pficon-resources-almost-empty`
* `pficon-resources-almost-full`
* `pficon-resources-full`

But no pficon for empty